tests/Tests.ClusterLauncher/Tests.ClusterLauncher.csproj

```diff
@@ -1,9 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>net462;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
   </PropertyGroup>
+  <PropertyGroup>
```

Comment:
Workaround for:

```
rror NU1004: The project's runtime identifiers have changed from. Project's runtime identifiers: , lock file's runtime identifiers win7-x86.The packages lock file is inconsistent with the project dependencies so restore can't be run in locked mode. Disable the RestoreLockedMode MSBuild property or pass an explicit --force-evaluate option to run restore to update the lock file.
```

This is a known long standing issue (see comments).

Another workaround that does not involve disabling lock files, would be this one:
https://github.com/NuGet/Home/issues/9195#issuecomment-1547896705